README.md

```diff
@@ -129,13 +132,16 @@ For framework- and provider- specific instructions, check out our docs:
 
 > Aikido is your no nonsense application security platform. One central system that scans your source code & cloud, shows you what vulnerabilities matter, and how to fix them - fast. So you can get back to building.
 
-Zen is a new product by Aikido. Built for developers to level up their security. While Aikido scans, get Zen for always-on protection.
+Zen improves Aikido's offering by providing you with security in production. Use the automatic generation of API specifications together with our DAST offering for even better API Security.
```

Comment:
Replace DAST with API scanning or something similar?

Reply:
I had no idea what DAST was and we want to bring these features to more engineers, so using industry terms will just confuse.

Reply:
```suggestion
Zen improves Aikido's offering by providing you with security in production. Use the automatic generation of API specifications together with our API scanning offering for even better API Security.
```